### DISCUSSION TITLE
Elastic scrolling on macOS

### DISCUSSION THREAD

**Discussion by luokuning** [zed-industries/zed#23910] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

As a macOS enthusiast, having an elastic effect (the rubber bounce effect when a page hits its top or bottom) can make scrolling more pleasant. Is it possible for Zed to easily add support for elastic scrolling in the macOS version?

**Comment by maxdeviant** [comment]
This is an interesting idea! I think this would need some more discussion as to whether this is something that makes sense in Zed.
